app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,44 @@
+class FoodsController < ApplicationController
+  before_action :authenticate_user!
+
+  # /foods
+  def index
+    @items = most_recent_added
+  end
+
+  # /foods (post)
+  def create
+    food = Food.new(foods_params)
+    food.user = current_user
+
+    if food.save
+      flash[:success] = 'Food has been created'
+    else
+      flash[:alert] = 'Error'
```

Comment:
[OPTIONAL] I think we could make this message more specific, to increase user experience. Something like "Check fields have proper data"